LICENSE

```diff
@@ -545,7 +545,7 @@ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
  * SUCH DAMAGE.
  *
- * The licence and distribution terms for any publically available version or
```

Comment:
This is text of "Original SSLeay License". We should keep this text even if it's typo.
https://www.openssl.org/source/license.html

Reply:
will do